app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListState.kt

```diff
@@ -5,12 +5,13 @@ import com.wire.android.ui.home.conversationslist.model.ConversationItem
 
 data class ConversationListState(
     val conversations: Map<ConversationFolder, List<ConversationItem>> = emptyMap(),
+    val hasNoConversations: Boolean = false,
+    val conversationSearchResult: Map<ConversationFolder, List<ConversationItem>> = emptyMap(),
```

Comment:
We need to hold somewhere alll the conversations to perform a filter on the conversation in this iteration, so I guess there is not really an alternative to do it ?

Reply:
@saleniuk  In the end I decided to keep it the way it is here, because when I was navigating back from "empty" search result very quickly there was a empty conversation screen for a second on the HomeScreen where you it tells you to click on "New" button to creata a conversation, this will be iterated pretty soon with a SQL query so there will be a need for a variable holding the search results I think